cpp/tests/groupby/count_scan_tests.cpp

```diff
@@ -54,8 +54,8 @@ TYPED_TEST(groupby_count_scan_test, basic)
   // clang-format on
 
   auto agg1 = cudf::make_count_aggregation<groupby_scan_aggregation>();
-  CUDF_EXPECT_THROW_MESSAGE(test_single_scan(keys, vals, expect_keys, expect_vals, std::move(agg1)),
-                            "Unsupported groupby scan aggregation");
+  EXPECT_THROW(test_single_scan(keys, vals, expect_keys, expect_vals, std::move(agg1)),
```

Comment:
A bunch of these cases would benefit from making the message into an inline code comment, so the test’s purpose is clear.

Reply:
Sure I can add those.

Reply:
Done.